E/Enzyme/build_tarballs.jl

```diff
@@ -8,13 +8,13 @@ include(joinpath(YGGDRASIL_DIR, "platforms", "llvm.jl"))
 name = "Enzyme"
 repo = "https://github.com/EnzymeAD/Enzyme.git"
 
-auto_version = "refs/tags/v0.0.46"
+auto_version = "refs/tags/v.0.0.47"
```

Comment:
@wsmoses  Wrong tag? 🙂

Reply:
@vchuravy

Reply:
I guess here we can do
```suggestion
auto_version = "refs/tags/v0.0.47"
```
but the tag may be wrong upstream?

Reply:
Yeah mistyped the tag.